packages/hub/node-tests/routes/scheduled-payments-test.ts

```diff
@@ -0,0 +1,561 @@
+import { PrismaClient } from '@prisma/client';
+import { calculateNextPayAt } from '../../utils/scheduled-payments';
+import { registry, setupHub } from '../helpers/server';
+import { setupStubWorkerClient } from '../helpers/stub-worker-client';
+
+class StubAuthenticationUtils {
+  validateAuthToken(encryptedAuthToken: string) {
+    return handleValidateAuthToken(encryptedAuthToken);
+  }
+}
+
+let stubUserAddress = '0x2f58630CA445Ab1a6DE2Bb9892AA2e1d60876C13';
+function handleValidateAuthToken(encryptedString: string) {
+  expect(encryptedString).to.equal('abc123--def456--ghi789');
+  return stubUserAddress;
+}
+
+describe('POST /api/scheduled-payments', async function () {
+  this.beforeEach(async function () {
+    registry(this).register('authentication-utils', StubAuthenticationUtils);
+  });
+
+  let { request } = setupHub(this);
+
+  it('returns a 401 if the auth token is invalid', async function () {
+    await request()
+      .post('/api/scheduled-payments')
+      .send({})
+      .set('Accept', 'application/vnd.api+json')
+      .set('Content-Type', 'application/vnd.api+json')
+      .expect(401)
+      .expect({
+        errors: [
+          {
+            status: '401',
+            title: 'No valid auth token',
+          },
+        ],
+      })
+      .expect('Content-Type', 'application/vnd.api+json');
+  });
+
+  it('responds with errors when attrs are missing', async function () {
+    await request()
+      .post('/api/scheduled-payments')
+      .send({
+        data: {
+          type: 'scheduled-payments',
+          attributes: {},
+        },
+      })
+      .set('Accept', 'application/vnd.api+json')
+      .set('Authorization', 'Bearer abc123--def456--ghi789')
+      .set('Content-Type', 'application/vnd.api+json')
+      .expect(422)
+      .expect({
+        errors: [
+          {
+            detail: 'is required',
```

Comment:
Maybe it’s redundant because it’s in `source.pointer` but the name `detail` suggests to me that it should say _what_ is required 🤓